app/main.py

```diff
@@ -1 +1,64 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = not self.is_hungry
+            return self.appetite
+
+        if self.is_hungry is False:
+            self.appetite = 0
+            return 0
+
+
+class Cat(Animal):
+    def __init__(
+            self,
+            name: str,
+            appetite: int = 3,
+            is_hungry: bool = True
```

Comment:
![image](https://github.com/mate-academy/py-feed-animals/assets/58142395/e832980a-60b2-4b5e-b327-25901d0d441c)

Reply:
i don't know how to change the arguments
if i make changes:

class Dog(Animal):
    def __init__(
            self,
            name: str,
            is_hungry: bool = True
    ) -> None:
        super().__init__(appetite=7) or (name, appetite=7, is_hungry)

it doesn't work